src/components/CourseModal.tsx

```diff
@@ -232,54 +224,25 @@ function CourseModal({ term, year }: DialogProps): JSX.Element {
           Add Course to {term}, Year {year}
         </DialogTitle>
         <DialogContent dividers>
-          <TextField
-            autoFocus
-            margin="dense"
-            id="code"
-            label="Course Code"
-            placeholder="CSCI134"
-            required
-            fullWidth
-            inputRef={codeRef}
-            autoComplete="off"
-          />
-          <TextField
-            autoFocus
-            margin="dense"
-            id="title"
-            label="Course Title"
-            placeholder="Operating Systems"
-            fullWidth
-            required
-            inputRef={titleRef}
-            autoComplete="off"
+          <Autocomplete
```

Comment:
Width seems off, we can check out the props it takes and see if the box width can be made bigger